### ISSUE TITLE
[Feature Request]  Vive wands default buttons

### ISSUE THREAD

**Issue by antispamtutoruser123** [praydog/UEVR#71]
Default Vive wands don't have face buttons assigned.
It would be nice if there was a way to change the default buttons globally through a file.  But if it can only be hardwired, a better default setup would be:  (These are the changes I immediately make in SteamVR for every game)

Right Grip = Right A Button
Right Menu = Right B Button
Left Grip = Left A Button
Left Menu = Left B Button
Right Menu (Long or double) = System Button
Trackpad Click Left = Joystick Click
Trackpad Click Right= Joystick Click

It can get more advanced from there but that's usually a good starting point.

**Comment by Hoshi82** [comment]
Yes, also for Index controller would be nice. A way to swap thumbsticks and having a default layout since there is no thumbrest button like with the touch controller. Having the d-pad on the touchpad of the Index left or right controller, depending on your handness.

**Comment by antispamtutoruser123** [comment]
I changed my mind, I think B/A should be the menu buttons.  Y/X should be grip buttons by default.
